pages/linux/trash.md

```diff
@@ -0,0 +1,23 @@
+# trash
+
+> A CLI trashcan/recycling bin.
+
+- Delete a file (send to trash):
+
+`trash-put {{path/to/file}}`
+
+- List files in trash:
+
+`trash-list`
+
+- Restore file from trash:
+
+`trash-restore`
```

Comment:
Doesn't that expect a parameter that says which is the file to be restored?

Reply:
Yes, but it seems kinda useless to me. 
Trash-list doesn't show the item id.
If you do not pass a parameter, it just lists all the files and waits for you to input a number, corresponding to the item to restore(actually shows numbers with trash-restore).

Reply:
Or actually no, it doesn't take parameters at all.

Reply:
Just running the command seems to be the only way to use it.

Reply:
Oh thats good then. We can leave it as it is.